cmd/monitor/import.go

```diff
@@ -24,36 +24,42 @@ import (
 	dd "github.com/tani-yu/dogleash/datadog"
 
 	"github.com/spf13/cobra"
-	"gopkg.in/zorkian/go-datadog-api.v2"
+	datadog "gopkg.in/zorkian/go-datadog-api.v2"
 )
 
-var inputPath string
+var target string
 
 // compute_checkCmd represents the compute_check command
 var monitorImportCmd = &cobra.Command{
 	Use:   "import",
-	Short: "json形式のファイルを読み込んでdatadogにdashboardを作成します",
+	Short: "Read json format file and create monitoring item on datadog",
 	Run: func(cmd *cobra.Command, args []string) {
 		cli, err := dd.NewDDClient()
 		if err != nil {
-			log.Fatalf("fatal: %s\n", err)
+			log.Fatalf("Failed to connect Datadog API server: %s\n", err)
 		}
 
-		raw, err := ioutil.ReadFile(inputPath)
-		if err != nil {
-			log.Fatalf("fatal: %s\n", err)
-			os.Exit(1)
+		if len(target) == 0 {
+			fmt.Println("Hello", target)
```

Comment:
You don't need to use `--target` option for this purpose, so you can remove `target` variable.